kratos/utilities/exact_mortar_segmentation_utility.cpp

```diff
@@ -29,7 +29,7 @@ bool ExactMortarIntegrationUtility<2, 2, false>::GetExactIntegration(
     )
 {
     // We take the geometry GP from the core
-    const double tolerance = 1.0e-6;  // std::numeric_limits<double>::epsilon();
+    const double tolerance = 1.0e3 * std::numeric_limits<double>::epsilon();
```

Comment:
everything still works?
I mean, it should be but one never knows with these tolerance things

Reply:
Yes, I checked